scripting/nexus-script-example/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'groovy'
```

Comment:
I wonder if this should use the Plugin DSL instead of `apply`.  The docs refer to `apply` as the legacy method of using plugins: https://docs.gradle.org/3.5/userguide/plugins.html